DataTests/BookmarkTests.swift

```diff
@@ -583,4 +598,27 @@ class BookmarkTests: CoreDataTestCase {
         XCTAssertNotNil(record.children)
         XCTAssert(record.children!.isEmpty)
     }
+    
+    // Bookmarklet validation
+    private func validateBookmarklet(title: String?, url: String?) -> Bool {
```

Comment:
This code is copied from `BookmarkDetailsView` if it changes there we might forget to update this test code.

Could we abstract it a bit? We could add an internal static method in `BookmarkDetailsView` and call it from both places(production and tests)

Reply:
Fixed.. but the code was so tightly coupled, I had to move a bunch of other code around as well otherwise the unit test gives a million errors trying to import.

Reply:
Have you tried doing `@testable import Client` and just call the static method from there?
`@testable` allow to call internal methods from other frameworks

Reply:
But I am not sure if calling that from Data framework tests on other hand..